components/automate-chef-io/content/docs/install.md

```diff
@@ -192,51 +192,88 @@ path = "/var/opt/chef-automate/backups"
-  # see https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3-client.html
-  # for full documentation on how to configure client settings on your
-  # Elasticsearch nodes
-  client = "<client name>"
-
-[global.v1.external.elasticsearch.backup.s3.settings]
-## The meaning of these settings is documented in the S3 Repository Plugin
-## documentation. See the following links:
-## https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3-repository.html
-
-## Backup repo settings
-# compress = false
-# server_side_encryption = false
-# buffer_size = "100mb"
-# canned_acl = "private"
-# storage_class = "standard"
-## Snapshot settings
-# max_snapshot_bytes_per_sec = "40mb"
-# max_restore_bytes_per_sec = "40mb"
-# chunk_size = "null"
-## S3 client settings
-# read_timeout = "50s"
-# max_retries = 3
-# use_throttle_retries = true
-# protocol = "https"
-```
+2. If you wish to use IAM authentication to provide your Elasticsearch nodes access to the S3 bucket, you must apply the appropriate IAM policy to each host system in the cluster.
+3. Configure each Elasticsearch node with a S3 client configuration containing the proper S3 endpoint, credentials, and other settings as [described in the Elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3-client.html).
+4. Enable S3 backups by adding the following settings to your config.toml:
+    ```toml
+    [global.v1.external.elasticsearch.backup]
+    enable = true
+    location = "s3"
+
+    [global.v1.external.elasticsearch.backup.s3]
+
+      # bucket (required): The name of the bucket
+      bucket = "<bucket name>"
+
+      # base_path (optional):  The path within the bucket where backups should be stored
+      # If base_path is not set, backups will be stored at the root of the bucket.
+      base_path = "<base path>"
+
+      # name of an s3 client configuration you create in your elasticsearch.yml
+      # see https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3-client.html
+      # for full documentation on how to configure client settings on your
+      # Elasticsearch nodes
+      client = "<client name>"
+
+    [global.v1.external.elasticsearch.backup.s3.settings]
+    ## The meaning of these settings is documented in the S3 Repository Plugin
+    ## documentation. See the following links:
+    ## https://www.elastic.co/guide/en/elasticsearch/plugins/current/repository-s3-repository.html
+
+    ## Backup repo settings
+    # compress = false
+    # server_side_encryption = false
+    # buffer_size = "100mb"
+    # canned_acl = "private"
+    # storage_class = "standard"
+    ## Snapshot settings
+    # max_snapshot_bytes_per_sec = "40mb"
+    # max_restore_bytes_per_sec = "40mb"
+    # chunk_size = "null"
+    ## S3 client settings
+    # read_timeout = "50s"
+    # max_retries = 3
+    # use_throttle_retries = true
+    # protocol = "https"
+    ```
+
+##### Backup to GCS
```

Comment:
@danielcbright Is this section the equivalent of "Backup Externally-Deployed Elasticsearch to GCS"? or is this not for externally-deployed Elasticsearch?

Reply:
also ok with this 💯